src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFHeaderLines.java

```diff
@@ -93,6 +93,7 @@ private static void addFilterLine(final VCFFilterHeaderLine line) {
         addFormatLine(new VCFFormatHeaderLine(PL_FOR_ALL_SNP_ALLELES_KEY, 10, VCFHeaderLineType.Integer, "Phred-scaled genotype likelihoods for all 4 possible bases regardless of whether there is statistical evidence for them. Ordering is always PL for AA AC CC GA GC GG TA TC TG TT."));
         addFormatLine(new VCFFormatHeaderLine(HAPLOTYPE_CALLER_PHASING_ID_KEY, 1, VCFHeaderLineType.String, "Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group"));
         addFormatLine(new VCFFormatHeaderLine(HAPLOTYPE_CALLER_PHASING_GT_KEY, 1, VCFHeaderLineType.String, "Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another"));
+        addFormatLine(new VCFFormatHeaderLine(HAPLOTYPE_CALLER_PHASING_SET_KEY, 1, VCFHeaderLineType.Integer, "Physical phasing set information, identifying the position of the first variant in the phasing group"));
```

Comment:
I have no idea why htsjdk has the PS constant, but no header line for it (https://github.com/samtools/htsjdk/blob/master/src/main/java/htsjdk/variant/vcf/VCFStandardHeaderLines.java)  I would rather add the header line in htsjdk, but I understand getting PRs through that repo can take a while.  Let's leave this in GATK and can you open a quick htsjdk PR?  We can refer to that once it's in.

Since "phase set" is how it's described in the VCF spec, I would rather the text say something like "Phase set information for physical phasing, identifying the position of the first variant in the phasing group"